docs/source/faqs-new.md

```diff
@@ -0,0 +1,40 @@
+(faqs-new)=
+# Frequently Asked Questions (FAQs)
+Below are some common questions and answers regarding our HPC cluster and its operation.
+
+::::{dropdown} My OOD job shuts down quickly before/during starting?
+There are two main things to check:
+1. Check that the memory available in `/home/$USER` is less than the quota limit.
+
+Run `du -shc .[^.]* * /home/$USER`, which should have output similar to the following:
+:::{code-block} shell
+[<username>@<host> ~]$  du -shc .[^.]* * /home/$USER/
+39M     .git
+106M    discovery-examples
+41K     README.md
+3.3M    software-installation
+147M    total
+:::
+
+Based on the output, we can check which files are consuming how much memory, and which files can be removed to free up memory.

```

Comment:
This is more applicable to the OOD error listed below. I think the FAQs would be better if we can aim the responses to the questions that the user often presents us.